src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -178,6 +178,63 @@ auto createForceModel( ParticleType particles, const double delta,
     return ForceModel<ModelType, DamageType, ThermalType, temp_type>(
         delta, K, temp, alpha, temp0 );
 }
+
+template <typename TemperatureType>
+struct ForceModel<PMB, Fracture, TemperatureDependent, TemperatureType>
+    : public ForceModel<PMB, Fracture, TemperatureIndependent>,
+      ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>
+{
+    using base_type = ForceModel<PMB, Fracture, TemperatureIndependent>;
+    using base_temperature_type = BaseTemperatureModel<TemperatureType>;
+    using base_model = typename base_type::base_model;
+    using fracture_type = typename base_type::fracture_type;
+    using thermal_type = typename base_temperature_type::thermal_type;
+
+    using base_type::c;
+    using base_type::delta;
+    using base_type::K;
+
+    using base_type::bond_break_coeff;
+    using base_type::G0;
+    using base_type::s0;
+
+    // Thermal coefficients
```

Comment:
I believe we used "// Thermal parameters" in the other PR

Reply:
Changed